src/main/scala/wdlTools/syntax/Antlr4Util.scala

```diff
@@ -18,14 +17,17 @@ import org.antlr.v4.runtime.{
   Recognizer,
   Token
 }
+
 import scala.jdk.CollectionConverters._
 import wdlTools.syntax
 import wdlTools.util.{Options, SourceCode, Verbosity}
 
 import scala.collection.mutable
+import scala.reflect.ClassTag
```

Comment:
Is it possible to avoid using reflection in this case?

Reply:
yes - I was trying to avoid some redundancy, but I guess it's not worth the use of reflection. I've gotten rid of it.